src/renderers/dom/fiber/ReactDOMFiberComponent.js

```diff
@@ -459,14 +460,16 @@ var ReactDOMFiberComponent = {
 
   mountComponent: function(
     workInProgress : Fiber,
+    tag : string,
     rootContainerElement : Element,
     props : Object,
     context : Object
   ) : Element {
+    // TODO:
     // validateDangerousTag(tag);
-    // workInProgress._tag = tag.toLowerCase();
+    // tag.toLowerCase(); Do we need to apply lower case only on non-custom elements?
```

Comment:
I'll do that in a follow up. I might need to do lower casing somewhere else because I don't have anywhere to store it. I figured we could do it when creating the fiber itself maybe.